### PR TITLE
Feat/sort filter, change profile image

### PR DESCRIPTION
**Pull request by bhavyaj12**
### Added sort by filter on user home feed page, other page posts sorted, added change profile image
- Added sort by filter component on user home feed page
- Sorted by latest posts in explore, bookmarks, user profile page
- Added profile images for all users, and edit profile image option in edit profile
- Db data additions and other minor refactoring

**Please check app is RESPONSIVE**
**NOTE: More posts data will be added soon after review**

### What not to check
- Search

### CODE REVIEW

File: src/components/FilterButtons.jsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import { useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { getPost, setSortBy } from "redux-reducers";
+import { FilterListOutlinedIcon } from "assets";
+
+const FilterButtons = () => {
+  const [showFilters, setShowFilters] = useState(false);
+  const { sortBy } = useSelector(getPost);
+  const dispatch = useDispatch(useDispatch);
```
Comment (vishalg8454) [inline src/components/FilterButtons.jsx]: Why passing useDispatch inside `useDispatch` ?

**Review by vishalg8454 — APPROVED** [review]
- Could not find sort button in user-profile and bookmark page.
- Padding can be reduced around Post Card in mobile view.
- can merge for now.

File: src/components/FilterButtons.jsx (hunk 1 above)
Comment (bhavyaj12) [reply src/components/FilterButtons.jsx]: Silly mistake, thanks!

Comment (bhavyaj12) [reply src/components/FilterButtons.jsx]: User profile and bookmarks are sorted by latest by default, sort option change is in only home page for now

Comment (vishalg8454) [reply src/components/FilterButtons.jsx]: > User profile and bookmarks are sorted by latest by default, sort option change is in only home page for now

Okay, I quoted from the second point in PR comment. You can add it later 👍

**Pull request merged**